tests/test_errorcallback.py

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+"""Tests for the error callback"""
+
+import unittest
+import unittest.mock
```

Comment:
The tests should run with Python version 2.7 and 3.3+. So, either make an import for mock:

    try:
        from unittest import mock
    except ImportError:
        import mock

or use pytest's mocking facilities.